docs/CONFIGURATION.md

```diff
@@ -5,9 +5,9 @@
 
 ## Configuration
 
-The following environmental variables can be set to configure behavior in Accounts.
+The following environment variables can be set to configure behavior in Accounts.
 
-| Environmental Variable | Description | Default |
+| environment Variable | Description | Default |
```

Comment:
This should be capitalized, I think the `replace` messed that up.

A couple of other lines in the file as well.